src/applications/CMakeLists.txt

```diff
@@ -3,25 +3,28 @@ if(WIN32)
   set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
   set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
 endif()
-file(COPY ${CMAKE_SOURCE_DIR}/../resources DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
```

Comment:
by file this means directory? 
is this better than ```execute_process``` and copy file approach? If it is, we should only use this, i have no issues with this. See #933 at the end of main CMakeLists file.